frontend/src/metabase-lib/v1/metadata/Database.ts

```diff
@@ -121,6 +121,10 @@ class Database {
     return Boolean(this.settings?.["database-enable-actions"]);
   }
 
+  hasTableEditingEnabled() {
+    return Boolean(this.settings?.["database-enable-table-editing"]);
+  }
```

Comment:
why do you need this in addition to the function in [frontend/src/metabase/databases/utils/settings.ts](https://github.com/metabase/metabase/pull/54990/files#diff-c8a6c1c8312c1868db8ce58ffbc92a35d7cb898f4a7b928c3f0ec9f6de20178c) ?